qt/widgets/common/inc/MantidQtWidgets/Common/ConfiguredAlgorithm.h

```diff
@@ -14,16 +14,19 @@ namespace MantidQt::API {
 class EXPORT_OPT_MANTIDQT_COMMON ConfiguredAlgorithm : public IConfiguredAlgorithm {
 public:
   ConfiguredAlgorithm(Mantid::API::IAlgorithm_sptr algorithm,
-                      std::unique_ptr<Mantid::API::IAlgorithmRuntimeProps> properties);
+                      std::unique_ptr<Mantid::API::IAlgorithmRuntimeProps> properties,
```

Comment:
Why this change has been introduced?

Reply:
This has been introduced to allow us to queue up the `Load` algorithm in the BatchAlgorithmRunner. This algorithm is slightly different to many others in that it will determine the appropriate load algorithm to use on runtime. This means we cannot validate the properties prior to the execution of the algorithm because if we do then it will only check if the properties exist on the `Load` algorithm, and not if they exist on the specialisation of the load algorithm it will actually choose. So adding this option allows us to skip the extra validation step in the BatchAlgorithmRunner at line 251